aws/aws.go

```diff
@@ -444,6 +449,108 @@ func deleteCFStacks(cfClients map[string]cfClient, rdsClients map[string]rdsClie
 	return errs
 }
 
+func disableDeleteProtection(rdsClient rdsClient, stackName string, dbInstanceId string, region string) error {
+	log.Infof("[AWS] Disabling DeletionProtection for DB instance: %s, region: %s", dbInstanceId, region)
+
+	_, err := rdsClient.ModifyDBInstance(&rds.ModifyDBInstanceInput{
+		DBInstanceIdentifier: &dbInstanceId,
+		DeletionProtection:   aws.Bool(false),
+	})
+	if err != nil && !strings.Contains(err.Error(), "DB instance not found") {
+		log.Errorf("[AWS] Skipping stack %s delete because failed to disable DeletionProtection on db instance: %s, err: %s", stackName, dbInstanceId, err)
+		return err
+	}
+	return nil
+}
+
+/**
+ * CloudFormation is not able to delete a VPC if it has dependencies, so we have to delete them manually.
+ */
+func deleteVpc(ec2Client ec2Client, stackName string, vpcId string, region string) error {
```

Comment:
This name not exactly covers what the function does. Could you please give a more self explanational name?

Reply:
Renamed to deleteVpcWithDependencies